ui/models.py

```diff
@@ -230,6 +230,20 @@ def transcoded_videos(self):
         return sorted(self.videofile_set.exclude(encoding=EncodingNames.ORIGINAL),
                       key=lambda x: EncodingNames.MP4.index(x.encoding) if x.encoding in EncodingNames.MP4 else 0)
 
+    @property
+    def download(self):
+        """
+        Return the most appropriate videofile for downloading.
+
+        Returns:
+            VideoFile: The videofile most appropriate for download (highest quality MP4 transcode or original upload)
+        """
+        files = sorted(self.videofile_set.exclude(encoding=EncodingNames.HLS),
+                       key=lambda x: EncodingNames.MP4.index(x.encoding) if x.encoding in EncodingNames.MP4 else 5)
```

Comment:
Can we replace `5` with `len(EncodingNames.MP4)` if that makes sense here?

Reply:
sure